test/typescript.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+var t = require('../')
+
+var spawn = require('child_process').spawn
+var path = require('path')
+var fs = require('fs')
+var fixdir = path.resolve(__dirname, 'ts')
+var ts = path.resolve(fixdir, 'test.ts')
+var run = require.resolve('../bin/run.js')
+var node = process.execPath
+
+function cleanup () {
+  try { fs.unlinkSync(ts) } catch (e) {}
+  try { fs.rmdirSync(fixdir) } catch (e) {}
+}
+
+t.test('setup', function (t) {
+  cleanup()
+  fs.mkdirSync(fixdir, '0755')
+  fs.writeFileSync(
+    ts,
+    'import * as tap from \'../../\';\n' +
```

Comment:
Style nit: prefer using `` ` `` for multiline strings.  Ie, instead of this, use something like:

```js
fs.writeFileSync(ts, `
import * as tap from '../../';
tap.test('TS Works', (t) => {
  t.equals(2, 1 + 1);
  t.end();
});
`)
```